src/components/Accordion/Accordion.test.tsx

```diff
@@ -9,16 +10,22 @@ import * as stories from './Accordion.stories';
 const { Default } = composeStories(stories);
 
 describe('<Accordion />', () => {
-  generateSnapshots(stories);
+  generateSnapshots(stories as StoryFile);
 
   it('should open and close Accordion panel clicking Accordion button', async () => {
     const user = userEvent.setup();
     render(<Default />);
     expect(screen.queryByTestId('accordion-panel')).not.toBeInTheDocument();
     const accordionButton = screen.getByTestId('accordion-button');
-    await user.click(accordionButton);
+
+    await act(async () => {
+      await user.click(accordionButton);
+    });
```

Comment:
Why do these events have to be wrapped in `act`, how'd you come to realize this?

Reply:
after upgrading dependencies, `yarn test` would output gigantic errors in console. what's strange is that for some of these user events, if you add the `act()` around them, it will complain that it's redundant 👎 

For whatever reason, not all `user.` calls need this, and on older versions of the libraries, none did. It doesnt make obvious sense which is which